src/more-infos/more-info-media_player.html

```diff
@@ -43,6 +43,10 @@
         margin-left: 10px;
       }
 
+      paper-dropdown-menu.source-input paper-listbox {
+        height: 96px;
```

Comment:
Why limit to 96px if there is enough space to show more?

Reply:
Well, the issue is that on mobile, the dropdown menu can't escape the paper-dialog-scrollable (?) and the overflow is cut-off. Limiting the size helps to make the menu work again, but this is probably not the best way to fix this. Any ideas?

Reply:
Don't have any ideas but we can't ship 96px height. We should find another solution.

Reply:
I just tried it. The scrollbox doesn't limit the dropdown. It extends all the way to the top of the screen.

Reply:
![img_5350](https://user-images.githubusercontent.com/4163868/30935517-3b6f9282-a3d1-11e7-9ea8-0420ea0f8a72.PNG)
Well it doesn't seem to work in Chrome/Safari/Home Assistant.app on iOS though.